Classes/common/HTTP/CDTURLSessionInterceptorContext.h

```diff
@@ -0,0 +1,39 @@
+//
+//  CDTURLSEssionFilterContext.h
+//  
+//
+//  Created by Rhys Short on 17/08/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "CDTMacros.h"
+
+@interface CDTURLSessionInterceptorContext : NSObject
+
+@property (nonnull, readonly, nonatomic, strong) NSMutableURLRequest *request;
```

Comment:
Apple place `nullable` and `nonnull` first. Example : 

```objc
@property (readonly, retain) NSOperationQueue *delegateQueue;
@property (nullable, readonly, retain) id <NSURLSessionDelegate> delegate;
@property (readonly, copy) NSURLSessionConfiguration *configuration;
``` 
Extract from NSURLSession.h, (c) Apple Inc.